AmericanMedicalAssociation/ama-style-guide-2
# EWL-9208 upgrade style-guide dependencies

## Ticket(s)
**Do not** submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**
- [N/A

**Jira Ticket**
- [EWL-9208: Styleguide Upgrade Alerts](https://issues.ama-assn.org/browse/EWL-9208)

## Description
Updates various dependencies within the styleguide repo.


## To Test
- with latest run 'gulp serve'
- confirm styleguide is accesible
- link-strleguides to local repo
- confirm styles compile when editing and that they are reflected on your local ama-d8 site.

## Visual Regressions
- N/A


## Relevant Screenshots/GIFs
-N/A


## Remaining Tasks
- N/A


## Additional Notes
- N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
